__test__/integrations/contact-loaders/helpers.test.js

```diff
@@ -0,0 +1,245 @@
+import { r } from "../../../src/server/models";
+import {
+  setupTest,
+  cleanupTest,
+  createStartedCampaign
+} from "../../test_helpers";
+import { finalizeContactLoad } from "../../../src/integrations/contact-loaders/helpers";
+
+const workersJobs = require("../../../src/workers/jobs");
+
+describe("contact-loaders/helpers", () => {
+  beforeAll(
+    async () => await setupTest(),
+    global.DATABASE_SETUP_TEARDOWN_TIMEOUT
+  );
+  afterAll(
+    async () => await cleanupTest(),
+    global.DATABASE_SETUP_TEARDOWN_TIMEOUT
+  );
+
+  describe("#finalizeContactLoad", () => {
+    let job;
+    let contacts;
+    let contactCells;
+    let campaign;
+
+    beforeEach(async () => {
+      const { testCampaign } = await createStartedCampaign();
+      campaign = testCampaign;
+    });
+
+    beforeEach(async () => {
+      job = {
+        campaign_id: campaign.id,
+        payload: 682951
+      };
+    });
+
+    beforeEach(async () => {
+      jest.restoreAllMocks();
+    });
+
+    beforeEach(async () => {
+      contacts = [
+        {
+          cell: "+13214028326",
+          custom_fields:
+            '{"CanvassFileRequestID":"1286","VanID":"6455083","Address":"627 Wizow Way, Lofaje, DE 89435","StreetAddress":"627 Wizow Way","City":"Lofaje","State":"DE","ZipOrPostal":"","County":"Suffolk","Employer":"","Occupation":"","Email":"","HomePhone":"","IsHomePhoneACellExchange":"","CellPhone":"(321) 402-8326","WorkPhone":"","IsWorkPhoneACellExchange":"","Phone":"(384) 984-5966","OptInPhone":"","OptInStatus":"","OptInPhoneType":"","CongressionalDistrict":"001","StateHouse":"004","StateSenate":"002","Party":"D","PollingLocation":"","PollingAddress":"","PollingCity":""}',
```

Comment:
These kind of custom field list seem better to keep inside the ngpvan.test.js file -- otherwise it might confuse someone that this is somehow part of 'core' Spoke.

Also, besides the general list/size of contact fields here are we testing something specific with these instances?  If so, it would be great to document in a comment what is 'special' about each record which might trip something up.